etsd/core/templates/core/edit_users.html

```diff
@@ -0,0 +1,39 @@
+{% extends "site_base.html" %}
+{% load render_table from django_tables2 %}
+{% load crispy_forms_tags %}
+
+
+{% block head_title %}Επιλογή χρηστών Υπηρεσίας{% endblock %}
```

Comment:
Please don't forget the translations. Also use a more generic title like "Change authority data" because they will also be able to change their email.